### ISSUE TITLE
Change blackfly optical from jpeg to tiff writer

### ISSUE THREAD

**Issue by jilavsky** [APS-USAXS/ipython-usaxs#439]
For BlackFly2 optical camera, collecting images for users on request. 
jpeg has 8 bit limitation, so we need to change to tiff file writer. Should be trivial.

**Comment by prjemian** [comment]
I anticipate a couple small bumps in switching the ophyd callback from jpeg
to Tiff.  Similar to those I had with the jpeg callback in the first place.
Nothing big.

**Comment by prjemian** [comment]
Also, see #434 for related matter.

**Comment by prjemian** [comment]
```python
RE(blackfly_optical.take_image())
```

We have an 8-bit image: [bluesky_0405.zip](https://github.com/APS-USAXS/ipython-usaxs/files/5598497/bluesky_0405.zip)

**Comment by prjemian** [comment]
FYI: `evince` can be used from linux to view these tiff files:

```
evince /share1/USAXS_data/testing/bluesky_0410.tif
```

**Comment by prjemian** [comment]
```python
%mov blackfly_optical.cam.pixel_format Mono16
```

This will produce 16-bit B/W images.

BUT, the detector is a color camera.  How to get color out of this beast?

**Comment by prjemian** [comment]
These are the possibilities:

```python
In [1]: blackfly_optical.cam.pixel_format.enum_strs
Out[1]: 
('Mono8',
 'YUV411',
 'YUV422',
 'YUV444',
 'RGB8',
 'Mono16',
 'Raw8',
 'Raw16',
 'Mono12',
 'Raw12',
 'BGR16')
```

**Comment by prjemian** [comment]
```python
In [4]: blackfly_optical.cam.pixel_format.put("BGR16")

In [5]: from instrument.devices.blackfly import AD_prime_plugin

In [6]: AD_prime_plugin(blackfly_optical, blackfly_optical.tiff1)

In [8]: blackfly_optical.tiff1.full_file_name.get()
Out[8]: '/mnt/share1/USAXS_data/testing/bluesky_0411.tif'

In [9]: !evince /share1/USAXS_data/testing/bluesky_0411.tif

```

Image is color with 8-bit per color (24-bit): [bluesky_0411.zip](https://github.com/APS-USAXS/ipython-usaxs/files/5598690/bluesky_0411.zip)

**Comment by prjemian** [comment]
This is working.

**Comment by prjemian** [comment]
This will be closed by PR #442
